client/src/components/SocialBar/SocialBar.test.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { i18n } from '@lingui/core';
+import { I18nProvider } from '@lingui/react';
+import { act, getByText, render } from '@testing-library/react';
 import SocialBar from './SocialBar';
+import { messages } from '../../locales/en/messages';
+import { messages as roMessages } from '../../locales/ro/messages';

```

Comment:
Would like to keep things consistent and have both `messages` imports keep their languages. So I'd suggest adding an `as enMessages` there 😅